android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt

```diff
@@ -137,6 +144,42 @@ class OkHttpTest {
     }
   }
 
+  data class HowsMySslResults(
+    val unknown_cipher_suite_supported: Boolean,
+    val beast_vuln: Boolean,
+    val session_ticket_supported: Boolean,
+    val tls_compression_supported: Boolean,
+    val ephemeral_keys_supported: Boolean,
+    val rating: String,
+    val tls_version: String,
+    val able_to_detect_n_minus_one_splitting: Boolean,
+    val insecure_cipher_suites: Map<String, List<String>>,
+    val given_cipher_suites: List<String>?
+  )
+
+  @Test
```

Comment:
Wanna `@Ignore` this so running our test suite by default doesn’t flake on connectivity?

Reply:
... I’d love to share tests like this between Android and JVM

Reply:
Yep, I'll follow up on that.